.env

```diff
@@ -35,4 +35,4 @@ ALLOW_MULTIPLE_ORGS=false
 
 DBT_PROJECT_DIR=.
 
-SCHEDULER_ENABLED=true
+SCHEDULER_ENABLED=true
```

Comment:
this is an unnecessary change. can your revert?

Reply:
sure

Reply:
sure